src/NimBLEUUID.cpp

```diff
@@ -156,6 +156,24 @@ NimBLEUUID::NimBLEUUID(ble_uuid128_t* uuid) {
 } // NimBLEUUID
 
 
+/**
+ * @brief Create a UUID from the 128bit value using hex parts instead of string, so BLEUUID("ebe0ccb0-7a0a-4b0c-8a1a-6ff2997da3a6") becomes BLEUUID(0xebe0ccb0, 0x7a0a, 0x4b0c, 0x8a1a6ff2997da3a6)
```

Comment:
Could we change this to:
```
 * @brief Create a UUID from the 128bit value using hex parts instead of string, 
 * instead of BLEUUID("ebe0ccb0-7a0a-4b0c-8a1a-6ff2997da3a6"), it becomes 
 * BLEUUID(0xebe0ccb0, 0x7a0a, 0x4b0c, 0x8a1a6ff2997da3a6)
```
for better presentation?

Reply:
Sure